server/datastore/mysql/software_titles.go

```diff
@@ -96,10 +96,6 @@ func (ds *Datastore) ListSoftwareTitles(
 		opt.ListOptions.OrderDirection = fleet.OrderDescending
 	}
 
-	if opt.AvailableForInstall && opt.VulnerableOnly {
-		return nil, 0, nil, fleet.NewInvalidArgumentError("query", "available_for_install and vulnerable can't be provided together")
-	}
-
```

Comment:
Tim, can you please provide more details as to what the problem is in #21376?

https://github.com/fleetdm/fleet/issues/21376